core/helpers/hardhat.ts

```diff
@@ -87,3 +98,38 @@ export const resetBlockchainFork = async function (
     const provider = await createWalletForRpc(signerPrivateKey, network);
     return provider;
 };
+
+export const addToBalance = async (
+    network: string,
+    walletAddress: string,
+    mkrAmount: BigNumber,
+    daiAmount: BigNumber
+) => {
```

Comment:
I would do it as two different functions `overwriteDaiBalance` and `overwriteMkrBalance` there is no need to combine them together